chrome/content/zotero/zoteroPane.js

```diff
@@ -1610,7 +1610,13 @@ var ZoteroPane = new function()
 			var prompt = (force && !fromMenu) ? false : toTrash;
 		}
 		else if (itemGroup.isCollection()) {
-			// In collection, only prompt if trashing
+			
+			// Don't prompt on remove when no top-level items are selected
```

Comment:
This comment is a little confusing. It's not not prompting on remove — it's not removing in the first place.

How about "Ignore unmodified action if only child items are selected"?